frontend/components/Contribute.vue

```diff
@@ -73,31 +72,66 @@
             </div>
           </div></b-list-group-item
         >
-        <b-list-group-item button @click="handleClick($event, 'placename')">
-          <div class="d-flex">
-            <div class="d-flex align-items-center pr-3">
-              <img
-                src="@/assets/images/add_area_icon_big.svg"
-                alt="Add a line"
-              />
-            </div>
-
-            <div>
-              <div class="contribute-list-group-title font-weight-bold">
-                Add a Placename
-              </div>
-              This option triggers drawing mode, where you can draw a line and
-              create a Placename
-            </div>
-          </div></b-list-group-item
-        >
       </b-list-group>
+      <div class="placename-btn-container">
+        <div class="media-add-btn" @click="validateArtist($event)">
+          <img
+            class="add-btn"
+            src="@/assets/images/artwork_icon.svg"
+            alt="Zoom In"
+          />
+          Upload Your Artwork <span>(I'm an Artist)</span>
+        </div>
+        <div
+          class="media-add-btn"
+          @click="handlePlaceClick($event, 'placename', 'Event')"
```

Comment:
This is handling clicks for PlaceNames, right? In my opinion, there's nothing inherently wrong about using long variable/function names such as `handlePlaceNameClick()`. I see that you've cut down the wording to keep it short, but it's really fine.

In this case, the function actually sounds like its purpose is to handle clicks on the places in the map. `handlePlaceClick()`

If a variable/function name being longer means it's a more accurate representation of what its purpose is, I think it's a good tradeoff.

@countable what do you think?

Reply:
I agree it's a trade-off. I'd probably go for the longer name personally since it's referenced only twice anyway.

Reply:
(or a handful of times)